### PR TITLE
Added option to include model information on asset labels.

### PR DESCRIPTION
**Pull request by jasonlshelton**
Cleaned up label page to fix skewed label alignment on last row per page.

### CODE REVIEW

**Comment by jasonlshelton** [comment]
Ok, got the right branch this time

File: database/migrations/2018_04_02_150700__labels_display_model.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('settings', function (Blueprint $table) {
+		$table->tinyInteger('labels_display_model')->default(1);
```
[the remark this answers is not in the document]
Comment (jasonlshelton) [reply database/migrations/2018_04_02_150700__labels_display_model.php]: I had just copied labels_display_tags. I'll change it

File: resources/views/hardware/labels.blade.php [hunk 2]
```diff
@@ -120,24 +126,30 @@
         @endif
         @if (($settings->labels_display_company_name=='1') && ($asset->company))
         <div class="pull-left">
-        	C: {{ $asset->company->name }}
```
[the remark this answers is not in the document]
Comment (jasonlshelton) [reply resources/views/hardware/labels.blade.php]: you da boss, I'll change it.

File: database/migrations/2018_04_02_150700__labels_display_model.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class LabelsDisplayCompanyName extends Migration
```
Comment (snipe) [inline database/migrations/2018_04_02_150700__labels_display_model.php]: FYI, this looks like some copypasta from a previous migration. This classname was used before, so it throws an error. I fixed it in https://github.com/snipe/snipe-it/commit/b4542d4d42ba6457c2c8c53b5cb7f28ae3c4259c

**Comment by snipe** [comment]
Can you look into #5935, @jasonlshelton? This may be a regression from this PR.

**Comment by snipe** [comment]
(Also I think we're not actually ever saving this preference anywhere? I can't find `labels_display_model` anywhere in the Settings controller.)

**Pull request merged**